packages/client/src/utils/stringReduction.ts

```diff
@@ -0,0 +1,3 @@
+export default function stringReduction(string: string, maxLength: number) {
```

Comment:
```suggestion
export default function stringShorten(string: string, maxLength: number) {
```
https://translate.google.com/?sl=ru&tl=en&text=%D1%83%D0%BA%D0%BE%D1%80%D0%BE%D1%82%D0%B8%D1%82%D1%8C&op=translate